lib/AssemblyUtil/FastaToAssembly.py

```diff
@@ -356,15 +364,16 @@ def _filter_contigs_by_length(self, fasta_file_path: Path, min_contig_length) ->
 
         return filtered_fasta_file_path
 
-    def _save_assembly_objects(self, workspace_id, assembly_names, ass_data):
+    def _save_assembly_objects(self, workspace_id, assembly_names, ass_data, ass_meta):
         print('Saving Assemblies to Workspace')
         sys.stdout.flush()
         ws_inputs = []
-        for assname, assdata_singular in zip(assembly_names, ass_data):
+        for assname, assdata_singular, assmeta_singular in zip(assembly_names, ass_data, ass_meta):
             ws_inputs.append({
                 'type': 'KBaseGenomeAnnotations.Assembly',  # This should really be versioned...
                 'data': assdata_singular,
-                'name': assname
+                'name': assname,
+                **({'meta': assmeta_singular} if assmeta_singular else {})
```

Comment:
```suggestion
                'meta': assmeta_singular
```
It seems like this should work, but maybe I'm missing something

Reply:
I think it will work but I did this to avoid modifying pre-existing tests that are not designed to handle 'meta' as input.

Reply:
You mean the mocking code? I def don't think we should make the code under test more complicated to avoid changing mocking inputs

Reply:
:+1: